src/components/Layout/Layout.scss

```diff
@@ -1,8 +1,23 @@
 .layout {
   display: grid;
-  grid-template-columns: 25% auto;
+  grid-template-columns: .25fr .25fr .25fr .25fr;
   max-width: 1440px;
   margin: 0 auto;
-  padding: 0 16px;
+  padding: 16px 16px;
   column-gap: 24px;
 }
+
+.layout__sidenav {
+  grid-column-start: 1;
+  grid-column-end: 1;
+}
+
+.layout__body {
+  grid-column-start: 2;
+  grid-column-end: 5;
+}
+
+.layout__full {
+  grid-column-start: 1;
+  grid-column-end: 5;
+}
```

Comment:
[nitpick] can you add a new line to the bottom of this file 
<img width="247" alt="Screen Shot 2020-11-04 at 1 33 22 PM" src="https://user-images.githubusercontent.com/13306885/98170173-64d2bd80-1ea2-11eb-9654-4a8971866003.png">
<img width="221" alt="Screen Shot 2020-11-04 at 1 34 23 PM" src="https://user-images.githubusercontent.com/13306885/98170218-74520680-1ea2-11eb-9972-389aeffcdcb5.png">